app/models/institution_builder.rb

```diff
@@ -522,6 +523,26 @@ def self.add_hcm(version_id)
       CautionFlag.build(version_id, HcmCautionFlag, caution_flag_clause)
     end
 
+    def self.add_poo_status_caution_flag(version_id)
```

Comment:
Make a builder in `app/utilities/institution_builder`, see other builders for naming and module/class structure